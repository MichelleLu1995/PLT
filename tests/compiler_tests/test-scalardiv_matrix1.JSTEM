def int main() {
	float[3][3] m1;
	float[3][3] m2;

	m1 = {% 10.2, 1.1, 2.2 | 4.5, 7.8, 4.4 | 6.8, 9.9, 3.2 %};

	m2 = m1 / 2.2;
	print_matrixf($$m2, 3, 3);

	return 0;
}
