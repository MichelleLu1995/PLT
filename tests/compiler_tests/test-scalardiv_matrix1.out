{ 4.636364, 0.500000, 1.000000
  2.045455, 3.545455, 2.000000
  3.090909, 4.500000, 1.454545 }
